tests/console/udisks2.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: 
+# Maintainer: João Walter Bruno Filho <bfilho@suse.com> 
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # install udisks2 package
+    zypper_call('in udisks2');
+
+    # Check disk status 
+    validate_script_output("udisksctl status", sub { m/Disk/s });
+
+    # Check if Udisks2 output is valid (contains a sections for Block and Partition Table).
+    validate_script_output("udisksctl info --block-device /dev/vda", sub { m/UDisks2.*Block.*PartitionTable/s} );
```

Comment:
what will happen if it  the test will be started with sda disk?

Reply:
It will certainly fail. I'd tested on local OpenQA instances and it only gave me vda, different than working locally with the command. I just considered it would work
I couldn't find a rule to get it from udisksctl status. The first entry doesn't seems to be always valid as the primary block device.

Reply:
Wouldn't be possible to check something like `/dev/?da` device?

Reply:
It took me a long time to do it, instead of just guessing, I used lsblk to check which devices are disk and then got those and checked with udisksctl. I also searched for partitions, but testing locally and on my instances it didn't always showed a valid partition (PartitionTable), so I also checked if the disk is partitionable (HintPartitionable: yes)  I'm not sure if this is the best way for it to be done, specially in perl.

Reply:
@jwbruno  please check https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/8b22b50f01708ddb424bfa16e467a3c7902277e8/tests/support_server/setup.pm#L421

Reply:
I was worried about testing a real disk, but I'm not sure if that's relevant to the test. Like a case like this (local machine):
# ls /dev/[sv]d[abc]
/dev/sda /dev/sdb /dev/sdc
# lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
sdc      8:32   0  1.8T  0 disk 
├─sdc1   8:33   0    2G  0 part [SWAP]
├─sdc2   8:34   0   40G  0 part /
└─sdc3   8:35   0  1.8T  0 part /home
sr0     11:0    1 1024M  0 rom  
where I would have different devices (i.e., card reader) with higher priority over the disk itself. 

I understand that the OpenQA is a more controlled environment, but since it varied so much between machines, I went to the one that worked for every case.
Either way would be fine for me.